baseapp-auth/baseapp_auth/rest_framework/users/serializers.py

```diff
@@ -14,13 +17,37 @@
 from .fields import AvatarField
 
 User = get_user_model()
+Profile = swapper.load_model("baseapp_profiles", "Profile")
 
 from baseapp_auth.password_validators import apply_password_validators
 
 
+class JWTProfileSerializer(serializers.ModelSerializer):
+    """
+    Serializes minimal profile data that will be attached to the JWT token as claim
+    """
+
+    url_path = serializers.SlugField(required=False, allow_null=True)
```

Comment:
`url_path` is not a slug field, its a `ForeignKey`, and in the current expected behavior can contain `/` in it. Would be very good to make tests related to this.

Reply:
@tsl-ps2 we need to check this

Reply:
I replaced the `SlugField` by a `SerializerMethodField`